src/app/admin/packingSlotsTable/PackingSlotsTable.tsx

```diff
@@ -165,6 +165,7 @@ const PackingSlotsTable: React.FC = () => {
                 setErrorMessage(
                     `Failed to add the packing slot. Log ID: ${insertPackingSlotError.logId}`
                 );
+                setRows((rows) => rows.slice(0, -1));
```

Comment:
Am I right in assuming the new slot will always be at the end of the array? The code which adds it does always add it at the end, the user can't reorder them until the creation is done, and I don't think there is any array sorting going on.

Reply:
I don't think so as far as I understand. When you click the button to add a new packing slot, it adds a new editable row to the table, which the user then fills in. And I think you have to do that by adding it to this array. So if we didn't add to the array then I'd imagine we couldn't fill out its details and save it.

Reply:
Ahh I see, that makes sense